app/services/sessions.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jwt-simple');
+const { moment } = require('../../config/moment');
+const { secret } = require('../../config').common.session;
+const logger = require('../logger');
+const { JWT_EXPIRATION_TIME } = require('../../config/constants');
+
+exports.generateToken = payload => {
+  try {
+    return jwt.encode(
+      {
+        ...payload,
+        exp: moment()
```

Comment:
Try to give it the expiration time without using another library, moment is a somewhat heavy library and using it only for this purpose does not seem very performant.

Reply:
In case it gets too complicated for you, I suggest you use this library [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken) , it is much better and it has easier handling to give it the expiration time.